server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -44,10 +44,13 @@ public class RwServer extends Thread {
         "application/json"
     ));
 
+    private static final int LOGGING_SHUTDOWN_TIMEOUT_SEC = 3;
```

Comment:
Is there a particular reasoning behind using 3 seconds? I would expect it to be much faster, but might it also be worth waiting a few more extra seconds to get the last logs in case it is a bit slow?

Reply:
No specific reason for just 3 seconds. I anticipate that logging generally be pretty quick, but we still need a timeout. The shutdown timeout of 20 seconds is too long. 3 seconds should probably be enough in 99.99% cases. But of course, we can make it whatever figure we like.

Reply:
There's no timeout anymore. This constant is removed.